core/components/mediamanager/model/mediamanager/classes/files.class.php

```diff
@@ -419,6 +427,46 @@ public function getFileHtml($fileId, $template)
             $i++;
         }
 
+        $bodyData['licensing.license_file_extensions'] = implode(', ', $sourceArray['licensing_file_allowed_extensions']);
```

Comment:
@sdrenth Maybe put an isset check for `$sourceArray['licensing_file_allowed_extensions']`?